examples/Animometer.cpp

```diff
@@ -49,7 +49,7 @@ void init() {
 
     queue = device.CreateQueueBuilder().GetResult();
     swapchain = GetSwapChain(device);
-    swapchain.Configure(nxt::TextureFormat::R8G8B8A8Unorm, 640, 480);
```

Comment:
Would it make sense to make the `Present` usage here implicit? It seems like it's going to be required no matter what.